graphql/database/missions/__tests__/utils.test.js

```diff
@@ -143,6 +143,7 @@ const mockDefaultMissionReturn = {
   tabCount: 258,
   squadMembers: [
     {
+      userId: 'cL5KcFKHd9fEU5C9Vstj3g4JAc73',
```

Comment:
this is mocking the return value we return to the client, where we hide userIds.  It looks like you're no longer using this value either

Reply:
I'm not sure I follow - I did amend `getCurrentUserMission` but it sounds like I shouldn't do that? I was adding it so I would have the userIds of squad members when calculating end of mission awards.

Reply:
actually that's' fine, graphql will still mask the user ids on calls to the FE so disregard